app/javascript/styles/mastodon/components.scss

```diff
@@ -7373,6 +7384,16 @@ noscript {
           width: 24px;
           height: 24px;
         }
+
+        &.copied {
+          border-color: $valid-value-color;
+        }
+      }
+
+      @media screen and (width <= 427px) {
+        .optional {
+          display: none;
+        }
```

Comment:
I think conditions on container sizes make more sense to also work properly on the advanced interface, but that's something that can be adjusted later.